<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConversionDialog</class>
 <widget class="QDialog" name="ConversionDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>350</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>350</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Tahoma</family>
    <pointsize>8</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Conversion Options</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="sizeConstraint">
    <enum>QLayout::SetFixedSize</enum>
   </property>
   <property name="horizontalSpacing">
    <number>6</number>
   </property>
   <property name="verticalSpacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QGroupBox" name="resolutionGroupBox">
     <property name="minimumSize">
      <size>
       <width>240</width>
       <height>80</height>
      </size>
     </property>
     <property name="title">
      <string>Resolution</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QCheckBox" name="convertResolutionCheckBox">
        <property name="toolTip">
         <string>Convert resolution</string>
        </property>
        <property name="text">
         <string>Convert &amp;resolution</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QFormLayout" name="formLayout_6">
        <item row="0" column="0">
         <widget class="QLabel" name="resolutionLabel">
          <property name="minimumSize">
           <size>
            <width>70</width>
            <height>20</height>
           </size>
          </property>
          <property name="text">
           <string>Resolution</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QComboBox" name="resolutionComboBox">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="minimumSize">
           <size>
            <width>150</width>
            <height>20</height>
           </size>
          </property>
          <property name="toolTip">
           <string>Select the target resolution</string>
          </property>
          <item>
           <property name="text">
            <string>NTSC (720x480)</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>PAL (720x576)</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>720p (1280x720)</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>1080p- (1440x1080)</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>1080p (1920x1080)</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="moveGroupBox">
     <property name="minimumSize">
      <size>
       <width>240</width>
       <height>50</height>
      </size>
     </property>
     <property name="title">
      <string>Move</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QCheckBox" name="applyMoveAllSettingsCheckBox">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="toolTip">
         <string>Apply settings for moving captions</string>
        </property>
        <property name="text">
         <string>Apply 'move all' settings</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QGroupBox" name="framerateGroupBox">
     <property name="minimumSize">
      <size>
       <width>240</width>
       <height>110</height>
      </size>
     </property>
     <property name="title">
      <string>Framerate</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QCheckBox" name="changeFrameRateCheckBox">
        <property name="toolTip">
         <string>Convert frame rate from FPS Source to FPS target</string>
        </property>
        <property name="text">
         <string>Change &amp;frame rate</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QFormLayout" name="formLayout_2">
        <item row="0" column="0">
         <widget class="QLabel" name="sourceFramerateLabel">
          <property name="minimumSize">
           <size>
            <width>70</width>
            <height>20</height>
           </size>
          </property>
          <property name="text">
           <string>FPS Source</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QComboBox" name="sourceFramerateComboBox">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="minimumSize">
           <size>
            <width>150</width>
            <height>20</height>
           </size>
          </property>
          <property name="toolTip">
           <string>Set the source frame rate (only needed for frame rate conversion)</string>
          </property>
          <property name="editable">
           <bool>true</bool>
          </property>
          <property name="currentIndex">
           <number>1</number>
          </property>
          <item>
           <property name="text">
            <string>23.975</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>23.976</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>24</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>25</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>29.97</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>50</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>59.94</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QFormLayout" name="formLayout_3">
        <item row="0" column="0">
         <widget class="QLabel" name="targetFramerateLabel">
          <property name="minimumSize">
           <size>
            <width>70</width>
            <height>20</height>
           </size>
          </property>
          <property name="text">
           <string>FPS Traget</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QComboBox" name="targetFramerateComboBox">
          <property name="minimumSize">
           <size>
            <width>150</width>
            <height>20</height>
           </size>
          </property>
          <property name="toolTip">
           <string>Set the target frame rate</string>
          </property>
          <property name="editable">
           <bool>true</bool>
          </property>
          <property name="currentIndex">
           <number>3</number>
          </property>
          <item>
           <property name="text">
            <string>23.975</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>23.976</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>24</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>25</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>29.97</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>50</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>59.94</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QGroupBox" name="timesGroupBox">
     <property name="minimumSize">
      <size>
       <width>240</width>
       <height>105</height>
      </size>
     </property>
     <property name="title">
      <string>Times</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QFormLayout" name="formLayout_5">
        <item row="0" column="0">
         <widget class="QLabel" name="delayLabel">
          <property name="minimumSize">
           <size>
            <width>70</width>
            <height>20</height>
           </size>
          </property>
          <property name="text">
           <string>Delay (ms)</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="delayLineEdit">
          <property name="toolTip">
           <string>Set global delay (in milliseconds) added to all timestamps</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QCheckBox" name="fixTooShortFramesCheckBox">
        <property name="toolTip">
         <string>Force a minimum display duration of 'Min Time'</string>
        </property>
        <property name="text">
         <string>Fix too &amp;short frames</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QFormLayout" name="formLayout_4">
        <item row="0" column="0">
         <widget class="QLabel" name="minTimeLabel">
          <property name="minimumSize">
           <size>
            <width>70</width>
            <height>20</height>
           </size>
          </property>
          <property name="text">
           <string>Min Time (ms)</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="minTimeLineEdit">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="toolTip">
           <string>Set minimum display time for a subtitle</string>
          </property>
          <property name="text">
           <string>520</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QGroupBox" name="forcedFlagsGroupBox">
     <property name="minimumSize">
      <size>
       <width>240</width>
       <height>50</height>
      </size>
     </property>
     <property name="title">
      <string>Global forced flags</string>
     </property>
     <widget class="QLabel" name="forceAllLabel">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>23</y>
        <width>70</width>
        <height>20</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>70</width>
        <height>20</height>
       </size>
      </property>
      <property name="text">
       <string>Force all</string>
      </property>
     </widget>
     <widget class="QComboBox" name="forceFlagsComboBox">
      <property name="geometry">
       <rect>
        <x>86</x>
        <y>23</y>
        <width>150</width>
        <height>20</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>150</width>
        <height>20</height>
       </size>
      </property>
      <property name="toolTip">
       <string>Select the target resolution</string>
      </property>
      <item>
       <property name="text">
        <string>keep</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>set all</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>clear all</string>
       </property>
      </item>
     </widget>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QGroupBox" name="defaultsGroupBox">
     <property name="minimumSize">
      <size>
       <width>240</width>
       <height>55</height>
      </size>
     </property>
     <property name="title">
      <string>Defaults</string>
     </property>
     <widget class="QPushButton" name="storeButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>23</y>
        <width>70</width>
        <height>22</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>Store current settings as default</string>
      </property>
      <property name="text">
       <string>St&amp;ore</string>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="restoreButton">
      <property name="geometry">
       <rect>
        <x>86</x>
        <y>23</y>
        <width>69</width>
        <height>22</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>Restore last default settings</string>
      </property>
      <property name="text">
       <string>R&amp;estore</string>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="resetButton">
      <property name="geometry">
       <rect>
        <x>161</x>
        <y>23</y>
        <width>69</width>
        <height>22</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>Reset defaults</string>
      </property>
      <property name="text">
       <string>Rese&amp;t</string>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
     </widget>
    </widget>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QFrame" name="buttonsPanel">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>9</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QPushButton" name="cancelButton">
        <property name="toolTip">
         <string>Lose all changes and use the default values</string>
        </property>
        <property name="text">
         <string>&amp;Cancel</string>
        </property>
        <property name="autoDefault">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>327</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="okButton">
        <property name="toolTip">
         <string>Use current values and continue</string>
        </property>
        <property name="text">
         <string>&amp;Ok</string>
        </property>
        <property name="autoDefault">
         <bool>false</bool>
        </property>
        <property name="default">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1" rowspan="2">
    <widget class="QGroupBox" name="scaleGroupBox">
     <property name="minimumSize">
      <size>
       <width>240</width>
       <height>105</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Tahoma</family>
      </font>
     </property>
     <property name="title">
      <string>Scale</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>10</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QCheckBox" name="scaleCheckBox">
        <property name="toolTip">
         <string>Allow free scaling of subtitles in X and Y direction</string>
        </property>
        <property name="text">
         <string>&amp;Apply free scaling</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QFormLayout" name="formLayout">
        <property name="verticalSpacing">
         <number>0</number>
        </property>
        <item row="0" column="0">
         <widget class="QLabel" name="scaleXLabel">
          <property name="minimumSize">
           <size>
            <width>70</width>
            <height>20</height>
           </size>
          </property>
          <property name="text">
           <string>Scale X</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="scaleXLineEdit">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="toolTip">
           <string>Set free scaling factor in X direction</string>
          </property>
          <property name="text">
           <string>1</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QFormLayout" name="formLayout_8">
        <property name="fieldGrowthPolicy">
         <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
        </property>
        <property name="verticalSpacing">
         <number>0</number>
        </property>
        <item row="0" column="1">
         <widget class="QLineEdit" name="scaleYLineEdit">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="toolTip">
           <string>Set free scaling factor in Y direction</string>
          </property>
          <property name="text">
           <string>1</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="scaleYLabel">
          <property name="minimumSize">
           <size>
            <width>70</width>
            <height>20</height>
           </size>
          </property>
          <property name="text">
           <string>Scale Y</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>10</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>okButton</tabstop>
  <tabstop>convertResolutionCheckBox</tabstop>
  <tabstop>resolutionComboBox</tabstop>
  <tabstop>applyMoveAllSettingsCheckBox</tabstop>
  <tabstop>changeFrameRateCheckBox</tabstop>
  <tabstop>sourceFramerateComboBox</tabstop>
  <tabstop>targetFramerateComboBox</tabstop>
  <tabstop>forceFlagsComboBox</tabstop>
  <tabstop>scaleCheckBox</tabstop>
  <tabstop>scaleXLineEdit</tabstop>
  <tabstop>scaleYLineEdit</tabstop>
  <tabstop>delayLineEdit</tabstop>
  <tabstop>fixTooShortFramesCheckBox</tabstop>
  <tabstop>minTimeLineEdit</tabstop>
  <tabstop>storeButton</tabstop>
  <tabstop>restoreButton</tabstop>
  <tabstop>resetButton</tabstop>
  <tabstop>cancelButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
